internal/nodes/avi_model_nodes_utils.go

```diff
@@ -0,0 +1,145 @@
+/* 
+* Copyright 2022-2023 VMware, Inc.
+* All Rights Reserved.
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*   http://www.apache.org/licenses/LICENSE-2.0
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+// Code generated by a tool; DO NOT EDIT.
+
+package nodes
+
+import (
+	"fmt"
+	"strings"
+
+	"google.golang.org/protobuf/proto"
+
+	// avimodels "github.com/vmware/alb-sdk/go/models"
```

Comment:
I think we can take out this commented import from the template present in the tool.

Reply:
Taken out from both ako code and template.